homework.txt

```diff
@@ -0,0 +1,38 @@
+Multiplication Tables//////////////////
+
+var i =0
+
+for (i=0; i<=10; i++){
+var result = i*9
+console.log(i+ "*9" + " =" + result );
+
+}/////////////////
+
+The Grade Assigner/////////
+
+
+for(i=60; i<=100; i++){
+function assignGrade(i){
+
+
+if(i>90){
+return "A";
+}
+else if(i>80){
+return "B";
+}
+else if(i>70){
+return "C";
+}
+else if(i>65){
+return "D";
+}else{
+return "F";
+}
+}
+ assignGrade(98);
```

Comment:
Good work here @asmaquraishi , my only note would be defining your function inside the loop. We don't wanna do that, the reason being our function will be redefined 0-i times. What we could do if we wanted to call that function on every iteration, 
```js
function assignGrade(i){
    if(i>90){
        return "A";
    }
    else if(i>80){
        return "B";
    }
    else if(i>70){
        return "C";
    }
    else if(i>65){
        return "D";
    } else {
        return "F";
    }
}

for(var i = 60; i <= 100; i++){
    assignGrade(i);
}
```

This way, we call the function all the times we want but not have to redefine it every time. Hope this helps!